src/test/java/org/trebol/jpa/services/conversion/PeopleConverterJpaServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package org.trebol.jpa.services.conversion;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.core.convert.ConversionService;
+import org.trebol.exceptions.BadInputException;
+import org.trebol.jpa.entities.Person;
+import org.trebol.pojo.PersonPojo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.trebol.constant.TestContants.ANY;
+
+@ExtendWith(MockitoExtension.class)
+class PeopleConverterJpaServiceImplTest {
+    @InjectMocks
+    private PeopleConverterJpaServiceImpl sut;
+
+    @Mock
+    private ConversionService conversionService;
+
+    private Person person;
+    private PersonPojo personPojo;
+
+    @BeforeEach
+    void beforeEach() {
+        person = new Person();
+        person.setId(1L);
+
+        personPojo = new PersonPojo();
+        personPojo.setId(1L);
+    }
+
+    @AfterEach
+    void afterEach() {
+        person = null;
+        personPojo = null;
+    }
+
+    @Test
+    void testApplyChangesToExistingEntity() throws BadInputException {
+        Person actual = sut.applyChangesToExistingEntity(personPojo, person);
```

Comment:
Again, not sure I see any need for these first two statements.
This test could do with a more accurate, verbose name such as `mutatesEntityEmailNamesAndPhonesFromPojo`